packages/graphviz/general-free.patch

```diff
@@ -0,0 +1,11 @@
+--- src/lib/sparse/general.h.orig	2016-04-01 08:13:25.222810595 -0700
++++ src/lib/sparse/general.h	2016-04-01 08:14:12.770312988 -0700
+@@ -14,6 +14,8 @@
+ #ifndef GENERAL_H
+ #define GENERAL_H
+ 
++void free(void *ptr);
```

Comment:
Honestly, I can't exactly remember why I needed to declare `free()`, but the compiler was complaining that it wasn't declared and it was easier to add than to figure out why.